### PR TITLE
Document SCIM again

### PR DESCRIPTION
**Pull request by fflorent**
Restore #434 fixing the issue with the SCIM API yaml.

### CODE REVIEW

**Review by hexaltation — CHANGES_REQUESTED** [review]
Just a nitpick in ENV variables descriptions

File: help/en/docs/install/scim.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+## The API
+
+The SCIM implementation is documented in the [Grist Rest API reference](/api/#tag/scim).
+
+## Enabling and configuring SCIM
+
+Below is a list of environment variables you may use to configure SCIM:
+
+- `GRIST_ENABLE_SCIM`: set its value to `true` so you enable the SCIM endpoints.
+- `GRIST_SCIM_USER`: (optional) give this variable the email address of an account to give it access to the SCIM endpoints.
```
[the remark this answers is not in the document]
Comment (fflorent) [reply help/en/docs/install/scim.md:42]: I don't agree, we should say `give [someone/something] access`. It's [distransitive](https://en.wiktionary.org/wiki/Appendix:Glossary#ditransitive) ([source](https://en.wiktionary.org/wiki/give))

[the remark this answers is not in the document]
Comment (fflorent) [reply help/en/docs/install/scim.md:42]: Also the wiktionary gives the example of the `give` verb for the ditransitivity:

> (of a verb) taking two objects, such as [give](https://en.wiktionary.org/wiki/give#English) in “Give me the ball” (where [me](https://en.wiktionary.org/wiki/me#English) is an [indirect object](https://en.wiktionary.org/wiki/Appendix:Glossary#indirect_object) and [the ball](https://en.wiktionary.org/wiki/ball#English) is a [direct object](https://en.wiktionary.org/wiki/Appendix:Glossary#direct_object)). Compare [intransitive](https://en.wiktionary.org/wiki/Appendix:Glossary#intransitive_verb), [transitive](https://en.wiktionary.org/wiki/Appendix:Glossary#transitive_verb), and [ambitransitive](https://en.wiktionary.org/wiki/Appendix:Glossary#ambitransitive_verb) verbs.

File: help/en/docs/install/scim.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+Also, SCIM allows you to deprovision users as soon as they leave. This ensures that they cannot log in without waiting their sessions to expire, which is a step toward complying with the GDPR if your organization is based in Europe.
+
+## The standard
+
+For more details on the SCIM standard, refer to the official IETF specifications: [RFC7643](https://www.rfc-editor.org/rfc/rfc7643) and [RFC7644](https://www.rfc-editor.org/rfc/rfc7644).
+
+## The API
+
+The SCIM implementation is documented in the [Grist Rest API reference](/api/#tag/scim).
```
Comment (jonathanperret) [inline help/en/docs/install/scim.md:35]:
```suggestion
The SCIM implementation is documented in the [Grist REST API reference](/api/#tag/scim).
```

File: help/en/docs/install/scim.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+The SCIM implementation is documented in the [Grist REST API reference](/api/#tag/scim).
+
+## Enabling and configuring SCIM
+
+Below is a list of environment variables you may use to configure SCIM:
+
+- `GRIST_ENABLE_SCIM`: set its value to `true` so you enable the SCIM endpoints.
+- `GRIST_SCIM_USER`: (optional) give this variable the email address of an account to give it access to the SCIM endpoints.
+
```
Comment (fflorent) [inline help/en/docs/install/scim.md:43]: Maybe explain that you can then use the endpoint by using the API key associated with the account above?

File: api/scim/users.yml [hunk 4]
```diff
@@ -0,0 +1,426 @@
+paths:
+  /scim/v2/Users:
+    get:
+      summary: Retrieve list of users
+      operationId: getUsers
+      tags:
+        - users
```
Comment (fflorent) [inline api/scim/users.yml:7]: Remove the `users` tag. Makes everything redundant in the API documentation.

Comment (fflorent) [reply api/scim/users.yml:7]: Also add the endpoint so it is referenced in the left menu
image

**Review by hexaltation — APPROVED** [review]
LGTM 🎅

**Comment by fflorent** [comment]
Thank you @hexaltation!

**Pull request merged**